components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -2512,6 +2512,16 @@ private User getSCIMUser(String userName, List<String> claimURIList, Map<String,
                         attributes.remove(SCIMConstants.UserSchemaConstants.ADDRESSES_URI);
                     }
 
+                    // Location URI is not available for users who created from the mgt console also location URI is not
+                    // tenant aware, so need to update the location URI according to the tenant.
+                    String locationURI = SCIMCommonUtils
+                            .getSCIMUserURL(attributes.get(SCIMConstants.CommonSchemaConstants.ID_URI));
```

Comment:
is 'getSCIMUserURL' method returning tenant aware URL?

Reply:
yes